Platform/Samsung/sm7150/a71.dsc

```diff
@@ -0,0 +1,29 @@
+[Defines]
+  VENDOR_NAME                    = Samsung
+  PLATFORM_NAME                  = a71
+  PLATFORM_GUID                  = 28f1a3bf-193a-47e3-a7b9-5a435eaab2ee
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sm7150/sm7150.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Samsung/sm7150/a71.fdf.inc
+
+!include Platform/Qualcomm/sm7150/sm7150.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE
+
+[PcdsFixedAtBuild.common]
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferWidth|1080
+  gQcomTokenSpaceGuid.PcdMipiFrameBufferHeight|2400
+  # gQcomTokenSpaceGuid.PcdMipiFrameBufferAddress|0x9c400000
+
+  # Simple Init
+  gSimpleInitTokenSpaceGuid.PcdGuiDefaultDPI|350
+
+  gRenegadePkgTokenSpaceGuid.PcdDeviceVendor|"Samsung"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceProduct|"Samsung Galaxy A71"
+  gRenegadePkgTokenSpaceGuid.PcdDeviceCodeName|"a71"
```

Comment:
"a71" is not a correct codename

Reply:
> "a71" is not a correct codename

No, it's correct. See https://storage.googleapis.com/play_public/supported_devices.html